tests/ui/dis/array_location_calculation.rs

```diff
@@ -0,0 +1,31 @@
+// build-pass
+// compile-flags: -C llvm-args=--disassemble-globals
+// normalize-stderr-test "OpCapability VulkanMemoryModel\n" -> ""
+// normalize-stderr-test "OpExtension .SPV_KHR_vulkan_memory_model.\n" -> ""
+// normalize-stderr-test "OpMemoryModel Logical Vulkan" -> "OpMemoryModel Logical Simple"
+// normalize-stderr-test "OpMemberName %12 0 .0.\n" -> ""
```

Comment:
What's up with this rewrite rule? It seems like it's probably hiding a bug that should be fixed.

Reply:
This is here because the ordering of the `OpMemberName`s gets altered between runs:

```
running 1 test
diff of stderr:

 OpCapability Float64
 OpCapability Int16
 OpCapability Int64
 OpCapability Int8
 OpCapability ShaderClockKHR
 OpCapability Shader
 OpExtension "SPV_KHR_shader_clock"
 OpMemoryModel Logical Simple
 OpEntryPoint Fragment %1 "main" %2 %3 %4 %5 %6 %7 %8 %9
 OpExecutionMode %1 OriginUpperLeft
 %10 = OpString "$OPSTRING_FILENAME/array_location_calculation.rs"
 OpMemberName %11 0 "x_axis"
 OpMemberName %11 1 "y_axis"
 OpMemberName %11 2 "z_axis"
 OpName %11 "spirv_std::glam::core::storage::Columns3<spirv_std::glam::XYZ<f32>>"
 OpMemberName %12 0 "0"
 OpName %12 "spirv_std::glam::Mat3"
 OpName %13 "array_location_calculation::main"
 OpName %2 "one"
 OpName %3 "two"
 OpName %4 "three"
 OpName %5 "four"
 OpName %6 "five"
 OpName %7 "six"
 OpName %8 "seven"
 OpName %9 "eight"
 OpMemberName %11 0 "x_axis"
 OpMemberName %11 1 "y_axis"
 OpMemberName %11 2 "z_axis"
-OpMemberName %12 0 "0"
-OpMemberName %12 0 "0"
 OpMemberName %11 0 "x_axis"
 OpMemberName %11 1 "y_axis"
 OpMemberName %11 2 "z_axis"
+OpMemberName %12 0 "0"
+OpMemberName %12 0 "0"

Reply:
Hm, that seems like a bug.